Makefile

```diff
@@ -0,0 +1,13 @@
+# Following go standards, the version of the code is associated to a tag in git
```

Comment:
is a Makefile really necessary ?

Reply:
It can be useful for anyone who would like to build the project without relying on docker

Reply:
fair enough